control/control.rnc

```diff
@@ -494,6 +494,20 @@ network = element network {
 
 # network
 
+# system_roles
+
+system_roles = element system_roles {
+    LIST,
+    system_role+
+}
+
+system_role = element system_role {
+    element id { text } &
+    partitioning? &
+    software?
```

Comment:
Yes, I've added only those sections that were explicitly requested in this Feature.
Obviously, in some layers of the implementation (like here in the rnc file) it would be easy to add more sections, but in actual code, software and partitioning have some differences already, and testing would be required...